Gemfile.lock

```diff
@@ -88,10 +88,7 @@ GEM
     jbuilder (2.7.0)
       activesupport (>= 4.2.0)
       multi_json (>= 1.2)
-    jquery-rails (4.3.1)
-      rails-dom-testing (>= 1, < 3)
-      railties (>= 4.2.0)
-      thor (>= 0.14, < 2.0)
+    json (2.1.0)
```

Comment:
Yes, I was getting issues when merging. It was very confusing, as there should have not been anything that changed in the Gemfile. I'm not sure how to proceed. I thought I had fixed it, but apparently not.

Reply:
Okay, so here is how each commit in this PR changed the Gemfile and Gemfile.lock:

a3bca1c (add bootstrap)
  Gemfile: added `bootstrap` and `jquery-rails` to the Gemfile
  Gemfile.lock: not updated

3bce9d4 (add basic templating)
  Gemfile: added `font-awesome-sass`
  Gemfile.lock: updated with info. for this and previous commit

71ba68d (merge upstream)
  Gemfile: no changes
  Gemfile.lock: no changes

c1a1d86
  Gemfile: no changes
  Gemfile.lock: spec for `json` added, spec for `jquery-rails` removed

The `json` gem is a dependency of the `simplecov` gem, which I added in PR #25, so that spec definitely needs to be in the gemfile.

I'm not sure what happened with the merge, but I think that the given the above analysis, we just need to add the spec for `jquery-rails` back in. As you might know, you should almost never make changes to `Gemfile.lock` by hand but I think this is one of those rare exceptions. I went ahead and committed this correction and pushed it to your branch - I hope you don't mind.

Just out of curiosity, can you tell me more about what happened when you merged changes from upstream? What files were conflicting? It's weird because [the merge commit](https://github.com/MilSpouseCoders/milspousecoders/pull/16/commits/71ba68d59face1535b23d2f4ccbf5f68266e6a84) says it was a clean merge. I thought if there were conflicts that were resolved, that would show up in the merge commit.

Another weird thing is that there is [a commit](https://github.com/MilSpouseCoders/milspousecoders/pull/16/commits/3bce9d4cf6a2f207016fe36c6cc08136fedb1847#diff-2) that shows that the `jquery-rails` spec was added to the lockfile, but there is no commit showing that it was removed, and yet that spec (for `jquery-rails`'s dependencies) wasn't in the lockfile on the head of your branch... so confusing 😕

Reply:
✔️